Sources/Networking/ImageDownloader.swift

```diff
@@ -164,7 +164,13 @@ open class ImageDownloader {
         authenticationChallengeResponder = self
         setupSessionHandler()
     }
-
+    
+    /// You could set the extra handler before a downloading task starts.
+    public weak var extraSessionDelegateHandler:URLSessionDataDelegate? {
```

Comment:
```suggestion
    public var extraSessionDelegateHandler:URLSessionDataDelegate? {
```

For a getter/setter, there should be no need to mark it as `weak`.

Reply:
好的，多谢！